src/controller/editingcontroller.js

```diff
@@ -31,6 +32,10 @@ import mix from '@ckeditor/ckeditor5-utils/src/mix';
  * including selection handling. It also creates {@link ~EditingController#view view document} which build a
  * browser-independent virtualization over the DOM elements. Editing controller also attach default converters.
  *
+ * Editing controller binds {@link module:engine/view/document~Document#roots view roots collection} to
```

Comment:
I don't think this is essential for the class overview. I believe this is an implementation detail. Even if, right now, this is an important behavior, for the person who wants to learn what `EditingController` is, binging roots is the least important information. I would move this comment to the place where binding is implemented.